### DISCUSSION TITLE
Customizing [openid_connect_generic_login_button]

### DISCUSSION THREAD

**Discussion by edgars** [oidc-wp/openid-connect-generic#546] [Q&A] [answered]
Hi people, 

Hi, to customize the [openid_connect_generic_login_button], I am using it on my page, but it looks like it is just a link. Is there a way to improve it? 

Thanks in advance

**Comment by timnolte** [comment, marked as answer]
You need to use CSS to style it according to your site. The markup is intentionally not use a `<button>` tag in order to keep it more flexible. It is surrounded by a `<div>`.

https://github.com/oidc-wp/openid-connect-generic/blob/0038ce797a2cc8ecdfcc6e67822f43e33de2409c/includes/openid-connect-generic-login-form.php#L153

**Comment by edgars** [comment]
Thanks! In the end, I got the [openid_connect_generic_auth_url], and tweaked according the current site's style.
